common/djangoapps/student/models.py

```diff
@@ -1196,6 +1196,13 @@ def users_enrolled_in(cls, course_id):
         )
 
     @classmethod
+    def enrolled_and_dropped_out_users(cls, course_id):
```

Comment:
Minor (but worth noting): These filter methods should be in a custom [`Manager`](https://docs.djangoproject.com/en/1.4/topics/db/managers/).